entrypoint.sh

```diff
@@ -9,8 +9,16 @@ install_zip_dependencies(){
 }
 
 publish_dependencies_as_layer(){
-	echo "Publishing dependencies as a layer..."
-	local result=$(aws lambda publish-layer-version --layer-name "${INPUT_LAMBDA_LAYER_ARN}" --zip-file fileb://dependencies.zip)
+	if [ "$INPUT_USE_S3" = true ]
+	then
+		echo "Uploading dependencies to S3..."
+		aws s3 cp dependencies.zip s3://"${INPUT_S3_BUCKET_NAME}"/dependencies.zip
+		echo "Publishing dependencies from S3 as a layer..."
+		local result=$(aws lambda publish-layer-version --layer-name "${INPUT_LAMBDA_LAYER_ARN}" --content S3Bucket="${INPUT_S3_BUCKET_NAME}",S3Key=dependencies.zip)
```

Comment:
I think the file key on S3 should be either configurable or derived from the lambda layer ARN; if the user has more than one lambda layer they would have to have different buckets for it which is not something we should force imo